botfront/imports/api/importExport/export.methods.js

```diff
@@ -12,30 +12,34 @@ import { generateErrorText } from './importExport.utils';
 
 
 if (Meteor.isServer) {
+    import { appLogger, addLoggingInterceptors } from '../../../server/logger';
+
+    const trainingAppLogger = appLogger.child({ fileName: 'export.methods.js' });
```

Comment:
Replace this with a function in 'logger' that abstracts that:

     const trainingAppLogger = getAppLogger(__filename)

Ottherwise, the choice of your logging library leaks all over your code...

Reply:
Thanks, I have made the change